binding/form_mapping.go

```diff
@@ -16,7 +16,15 @@ import (
 	"github.com/gin-gonic/gin/internal/json"
 )
 
-var errUnknownType = errors.New("unknown type")
+var (
+	errUnknownType = errors.New("unknown type")
+
+	// ErrConvertMapStringSlice covert to map[string][]string
```

Comment:
replace `covert to map[string][]string` with `can not covert to map[string][]string`

Reply:
This is not what you said, this is the error returned, as a developer using GIN can know the error of the underlying transformation, rather than what you said not to let others know, which is a bad design

Reply:
```go
// ErrConvertMapStringSlice covert to map[string][]string
// ErrConvertToMapString can not convert to map[string]string
```

I don't know why the format can't be the same between `ErrConvertMapStringSlice` and `ErrConvertToMapString`?

Reply:
I mean that comment message.

```go
errors.New("can not convert to map of strings")
```

comment message:

```go
// can not convert to map[string]string
```

but

```go
errors.New("can not convert to map slices of strings")
```

comment message:

```go
// covert to map[string][]string
```

not 

```go
// can not convert to map[string][]string
```

Reply:
OK, if there are two errors here, they should be separated. As for the comment information you said, you can modify it appropriately. For the function point here, I hope the caller knows why the error occurred, instead of returning as before.

Reply:
> I mean that comment message.
> 
> ```go
> errors.New("can not convert to map of strings")
> ```
> 
> comment message:
> 
> ```go
> // can not convert to map[string]string
> ```
> 
> but
> 
> ```go
> errors.New("can not convert to map slices of strings")
> ```
> 
> comment message:
> 
> ```go
> // covert to map[string][]string
> ```
> 
> not
> 
> ```go
> // can not convert to map[string][]string
> ```

You can change the notes here. Thank you very much

Reply:
I just want to change the comment

```diff
-// ErrConvertMapStringSlice covert to map[string][]string
+// ErrConvertMapStringSlice can not covert to map[string][]string
```